scripts/filename-checker

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
The file should be in scripts/dev. Then you probably need another script in scripts/build/ which runs the filename-checker for every file, similar to what the scripts/dev/reformat- are doing.